src/main/java/club/sk1er/patcher/screen/render/overlay/ImagePreview.java

```diff
@@ -148,11 +154,33 @@ private void loadUrl(String url) {
             connection = (HttpURLConnection) u.openConnection();
             connection.setRequestMethod("GET");
             connection.setUseCaches(true);
+            connection.setInstanceFollowRedirects(true);
             connection.addRequestProperty("User-Agent", "Patcher Image Previewer");
             connection.setReadTimeout(15000);
             connection.setConnectTimeout(15000);
             connection.setDoOutput(true);
 
+            if (connection.getHeaderField("Content-Type").contains("text/html")) {
+                String body = IOUtils.toString(connection.getInputStream());
```

Comment:
Shouldn't this be in the try block? I would assume the other `connection.getInputStream()` is in one for a reason.

Reply:
I moved the try block to contain it all and remove the duplicate getInputStream calls